### DISCUSSION TITLE
Usage of Submit Details in Profile

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-Ed/Community#345] [Sunbird ED configuration discussion] [answered]
Hello All ,

I am trying to understand the use of Submit Details button on the profile of the User.

As per my understanding there is a State login provided by Diksha to the teacher and the other one they are creating so that they can consume the courses on the portal. And then the teacher can share progress with the organization they are associated with .

I have a doubt regarding why the teacher cannot consume the course from state login itself.

![sub-det-btn](https://user-images.githubusercontent.com/65657373/211295964-99f16f95-443c-43a2-a56e-8b9c6fc420d0.png)

cc: @komalm @renuka-S @vinod303

**Comment by Krishnaj20** [comment]
@kameshbhr @mmanimegalai @vinod303 - can you respond to this query on how this is used by DIKSHA?

**Comment by kameshbhr** [comment]
I am not clear on the question. Please write to DIKSHA support team for any queries related to DIKSHA.
Submit Details button is to allow users to share the profile details to administrators of a specific organization (typically to the org the user belongs to). This has nothing to do with who can consume courses.

**Reply by Krishnaj20** [reply]
@Xitija - Is your query answered?

**Reply by Xitija** [reply]
I have not fully understood the usage of this functionality yet.

**Reply by Krishnaj20** [reply]
@Xitija - Teachers can consume DIKSHA content using their state logins via SSO.  Can you explain what is your query? Pls note, as mentioned by Kamesh, if it is a DIKSHA query then it has to be directed to DIKSHA support id - Vinod can help you with this.

**Reply by Xitija** [reply]
If the teacher can consume the content on the platform directly why cant the organisation associated with it see the progress of that directly ? What is the reason that they have to share the  progress explicitly ?

**Comment by Renuka-S** [comment, marked as answer]
@kameshbhr @Krishnaj20 To simply put we are looking for the use of the "submit details" option on the user profile. Which details can be submitted? which roles can view this? We have observed not all users view this. How this can be configured? and as @kameshbhr mentioned these details are submitted for review to the admins. Can you help us understand the admin workflow for this where can view and approve these details?

cc: @Xitija

**Reply by Krishnaj20** [reply]
@Xitija @Renuka-S - I'm not sure I understand the question. @Xitija is talking about DIKSHA teachers - is this about DIKSHA teachers or whom?

**Reply by Renuka-S** [reply]
@Krishnaj20 I have elaborated on our question above. could you please check?

We are looking for the usage of the "Submit details" option in SUNBIRD, this isn't related to Diksha.

**Reply by Krishnaj20** [reply]
'Submit details' allows users who belong to the default tenant and not to any specific tenant share their details with any other tenant admin in the instance. If the user already belongs to a tenant then this option doesn't show up. 
The fields in the form can configured. @reshmi-nair can help with the form configurations, if you have any queries. 
The data shared by the user is made available to the org admin via an exhaust which runs overnight. @reshmi-nair can help with any queries about this exhaust and the fields that are captured in this exhaust. 

@reshmi-nair @rhwarrier - I couldn't find the documentation for the exhaust in the lern microsite. Please point us to the relevant documentation.

**Reply by rhwarrier** [reply]
Let me try elaborate : 

1. An organisation may have a set of users using Diksha for a purpose (say a batch of teachers from a state taking a course)
2. The state may want to track their usage on the platform
3. The end user enables this by giving consent to the platform to share their data with the said Org (state, in this case)

![Screenshot 2023-01-19 at 9 40 06 AM](https://user-images.githubusercontent.com/24287865/213353216-71f31e96-1e71-4624-80d7-d0469ce0c0ab.png)

4. The user gives their consent to share their info by submitting the form via the profile page
5. The form allows the user to submit a few PII details that the org associates with the user - for example, a teacher can submit an ID that the state may have assigned to them, their official email ID / phone number that they have been assigned by the state - these are identifiers assigned to the users by the org/ state, and have nothing to do with Diksha.
6. The user can choose which org they want to share their details with - for example, a user from an NGO may choose to share their details with the org created for the NGO on the platform. (as of now, a user can share their details only with one Org)
7. These identifiers become relevant when the org tries to map usage data from Diksha (say an anonymised course consumption report that the course system provides) with the PII data provided by the user


The anonymised Course progress exhaust has only the users Diksha UUID, in addition to course progress details.

PII report provided to Org Admin:
Name,Diksha UUID,State,District,Block,Cluster,School Name,School UDISE ID,State provided ext. ID,Profile Email,Profile Phone number,Org Phone,Org Email ID,User Type,User-Sub Type,Root Org of user
(State provided ext ID, Org phone and Org email ID are submitted by the user via the form)

As you can see, mapping the users records from these two reports together wil allow the admin visibility into which user from the org has made what progress with the course. 

The user PII report is shared at 2 levels - 
1. users who take a course can choose to share their PII with course creators/ mentors 
2. users can choose to share their PII with the admin of any Org they wish to share their details with.

The course level user PII exhaust is documented on the following page : https://lern.sunbird.org/learn/product-and-developer-guide/batch-service/reports

Looks like we need to add the admin level PII report to the microsite 
@reshmi-nair please note. 

Hope this helps.

@Xitija @Renuka-S   It'll also help if you could articulate your use case - we could see if this feature can prove to be of use in your context. The Diksha team can help with further details on how this is being used in the context of states wanting to track course consumption by their teachers.

**Reply by reshmi-nair** [reply]
@Xitija @Renuka-S Pls find the consent report details in lern microsite: [Reports ](https://lern.sunbird.org/learn/product-and-developer-guide/user-and-org-service/reports)
